concepts/strings/introduction.md

```diff
@@ -0,0 +1,220 @@
+# Introduction
```

Comment:
The introduction is meant to be a bit more of a summary, so should be a bit shorter than about.md.

Reply:
I summarized the information in a table.

Reply:
Awesome, thats a useful reference :)

Reply:
I think this is a bit too summarized, this is the entry point for the students, so I would at least show a string, a multiline string, a couple of common functions spanning a few different sections (concat, `++`, split, join, replace, length, toInt, contains, map, or something like that?)